### PR TITLE
Add issue templates for bug reports and feature requests

### PR DESCRIPTION
**Pull request by gabbifish**
This PR addresses #250 by adding a bug_report.md template and a feature_request.md template. I primarily drew the bug report template from golang's template, which is good at asking descriptive boilerplate questions to kickstart debugging/assessment.

Let me know if there are any boilerplate questions I might be missing.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+</pre>
+
+#### Does this issue reproduce with the latest release?
+
+
+
+#### What environment are you using?
+* operating system:
+* rustc version:
```
Comment (ashleymichal) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
* rustc version:
* node version:
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+### 💡 Feature request
+Brief explanation of the requested feature.
```
Comment (ashleymichal) [inline .github/ISSUE_TEMPLATE/feature_request.md]: i think also "what problem this would solve" might be a nice ask.

**Comment by ashleymichal** [comment]
do issue templates include any sort of metadata for things like labels? I know we discussed including `user report` by default and asking members to remove it on their own issues.

**Comment by gabbifish** [comment]
@ashleymichal github does let you include metadata for labels! I'll add this in another commit: https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository

**Pull request merged**